RELEASE_NOTES.md

```diff
@@ -1,5 +1,81 @@
 # Release Notes
 
+## 5.4.0
+### Features
+- Initial support for artifacts has been added through the `podman artifact` suite of commands, including `add`, `inspect`, `ls`, `pull`, `push`, and `rm`. This support is very early, and the command line interface for these tools has not been finalized. We welcome feedback on the new artifact experience through our issue tracker!
```

Comment:
you imply this is dev ... i wonder if we should explicitly say so ?

Reply:
Maybe "A preview our Podman's support for artifacts has been added..."?